src/test/techdebt.test.ts

```diff
@@ -34,6 +35,11 @@ describe('tech debt', function () {
             semver.lt(minVscode, '1.56.0'),
             'remove AsyncLocalStorage polyfill used in `spans.ts` if Cloud9 is on node 14+'
         )
+
+        assert.ok(
+            semver.lt(minVscode, '1.64.0'),
+            `remove ${quickPickSeparator.name}() since the vscode API now supports it`
```

Comment:
there is alrady a test on line 32. it needs to be updated

Reply:
Since https://github.com/aws/aws-toolkit-vscode/pull/3328 was merged we don't need this test now.